app-core/src/main/java/com/mercury/platform/shared/messageparser/PoeTradeItemKoreanParser.java

```diff
@@ -0,0 +1,39 @@
+package com.mercury.platform.shared.messageparser;
+
+import com.mercury.platform.shared.entity.message.ItemTradeNotificationDescriptor;
+import com.mercury.platform.shared.entity.message.NotificationDescriptor;
+import com.mercury.platform.shared.entity.message.NotificationType;
+
+import java.util.regex.Matcher;
+
+class PoeTradeItemKoreanParser extends BaseRegexParser {
+
+    /** Any text between {@code \Q} and {@code \E} will be matched as raw text (escaped). */
+    private final static String poeTradeKoreanRegex = "^\\s*(?<name>.*?): \\Q안녕하세요, \\E(?<league>.*)\\Q(보관함 탭 \"\\E(?<stashtab>[^\"]*)\\Q\", 위치: 왼쪽 \\E(?<left>\\d+)\\Q, 상단 \\E(?<top>\\d+)\\Q)에 \\E(?<price>[\\d.]*)\\s+(?<currency>.*)\\Q(으)로 올려놓은 \\E(?<item>.*)\\Q(을)를 구매하고 싶습니다\\E";
```

Comment:
Can also mention that the `.*?` in `name` is a non-greedy dotall to stop at the first `:` instead of the last.